src/utils/ethereum/eip/eip681_test.cljs

```diff
@@ -111,9 +111,9 @@
                                                            :name "Status Network Token"
                                                            :symbol :SNT
                                                            :decimals 18}}
-   :goerli  {"0x3d6afaa395c31fcd391fe3d562e75fe9e8ec7e6a"
+   :sepolia {"0xE452027cdEF746c7Cd3DB31CB700428b16cD8E51"
```

Comment:
nit: You can move this to a constant. If Sepolia testnet becomes obsolete in the future, we can just update that constanta.